companies/513947119.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "iCount",
+    "description": "Everything a business needs and more. Finally, a system that can meet all of your business’s needs from the basic to the most advanced.",
+    "categoryId": 10,
+    "size": "s",
+    "logoFile": "logos/icount_logo.webp",
```

Comment:
add a 200x200px webp, png or jpeg file as part of your submission

Reply:
I already uploaded, maybe you missed it?
![image](https://github.com/mluggy/techmap/assets/1343001/15eaef66-ea6e-4306-b86e-a321b2dc8ce3)